eclass/systemd.eclass

```diff
@@ -405,6 +461,29 @@ systemd_is_booted() {
 	return ${ret}
 }
 
+# @FUNCTION: systemd_tmpfiles_create
+# @USAGE: <tmpfilesd> ...
+# @DESCRIPTION:
+# Deprecated in favor of tmpfiles.eclass.
+#
+# Invokes systemd-tmpfiles --create with given arguments.
+# Does nothing if ROOT != / or systemd-tmpfiles is not in PATH.
+# This function should be called from pkg_postinst.
+#
+# Generally, this function should be called with the names of any tmpfiles
+# fragments which have been installed, either by the build system or by a
+# previous call to systemd_dotmpfilesd. This ensures that any tmpfiles are
+# created without the need to reboot the system.
+systemd_tmpfiles_create() {
```

Comment:
Why should we bring back the function `systemd_tmpfiles_create()`?
Looks like the function is not used anywhere, right?

Reply:
Hm, I just brought back everything related to tmpfiles. But yeah, this one is actually unused, I'll drop it.

Reply:
Removed.